docs/foundations/typography/scale-and-rhythm/index.md

```diff
@@ -0,0 +1,272 @@
+---
+title: Scale and rhythm
+heading: Typography
+sidenavTitle: Typography
+layout: layouts/pages/has-toc.njk
+permalink: /foundations/typography/scale-and-rhythm/index.html
+tags:
+  - typography
+subnav:
+  collection: sortedTypography
+  order: 5
+---
+
+<link data-helmet rel="stylesheet" href="/assets/packages/@rhds/elements/elements/rh-table/rh-table-lightdom.css">
+
+<script type="module" data-helmet>
+  import '@rhds/elements/rh-table/rh-table.js';
+</script>
+
+<style data-helmet>
+  rh-table {
+    & td, & th { width: 20%; }
+    & td {
+      &:has(s) { color: var(--rh-color-text-brand); }
+      & span { color: var(--rh-color-green-60); }
+      & s { color: var(--rh-color-gray-40); }
+      & :is(s, span) { display: block; }
+    }
+  }
+</style>
+
+
+## Overview
+
+Our type scale features a range of text sizes and weights designed to support lots of content and user needs.
+
+For always up-to-date type specs, go to the [Tokens][tokens] section.
+
+### Base text size
+
+We use a base text size of 16px or 1.0rem.
+
+## Desktop scale
+
+### Extra large headings
+
+These are Expressive type heading sizes.
+
+To learn more about how to use these, go to the [Choosing type][choosingtype] page.
+
+<rh-table>
+
+| Name  | Font family            | Weight   | Size        | Line height (1.1) |
+| ----- | ---------------------- | -------- | ----------- | ----------------- |
+| 5xl   | Display, Text, Mono    | Multiple | 96 (6.0rem) | 105.6             |
+| 4xl   | Display, Text, Mono    | Multiple | 80 (5.0rem) | 88                |
+| 3xl   | Display, Text, Mono    | Multiple | 64 (4.0rem) | 70.4              |
+
+</rh-table>
+
```

Comment:
I wonder if it's worth linking to the specific tokens below each table? Could be a CTA?

Reply:
It might feel a little repetitive to add a CTA To the typography tokens page under each table. There was an inline link in the "Overview" section of this page, but I changed a sentence to say, "Use typography tokens to implement these styles," and added a CTA below.